cmd/api/src/daemons/api/toolapi/api.go

```diff
@@ -66,6 +67,17 @@ func NewDaemon[DBType database.Database](ctx context.Context, connections bootst
 	router.Get("/pg-migration/status", pgMigrator.MigrationStatus)
 	router.Put("/pg-migration/cancel", pgMigrator.MigrationCancel)
 
+	// Allow query of datapipe status for infrastructure tooling
+	router.Get("/datapipe/status", func(w http.ResponseWriter, r *http.Request) {
+		if dpStatus, err := connections.RDMS.GetDatapipeStatus(ctx); err != nil {
+			api.WriteJSONResponse(r.Context(), map[string]any{
+				"error": "Datapipe status unavailable",
+			}, http.StatusInternalServerError, w)
```

Comment:
`HandleDatabaseError` util might be able to be leveraged here

Reply:
ack; Good catch @mistahj67 !